lumps/dehacked.lmp

```diff
@@ -173,10 +173,10 @@ CC_DEMON = flesh worm
 CC_LOST = deadflare
 CC_CACO = orb monster
 CC_HELL = pain bringer
-CC_BARON = pain lord
+CC_BARON = pestmeister 
```

Comment:
Not sure about "pestmeister" as a monster name. "Pest" usually implies something small, verminous and irritating, like a fly or a mouse. It's also nice to have some kind of commonality of name between the Hell Knight / Baron of Hell names.